python/plugins/MetaSearch/util.py

```diff
@@ -167,3 +166,10 @@ def clean_ows_url(url):
         url.setQuery(query_string)
 
     return url.toString()
+
+
+def log_message(message, level='Info'):
+    """helper function to emit logging messages"""
+
+    message_translated = QCoreApplication.translate('MetaSearch', message)
```

Comment:
This won't actually work for translations, as the strings to translate can't be collected in advance via this approach. You need to translate the strings before calling log_message, and then using them directly here.

Reply:
updated.